pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -90,41 +85,28 @@ var updateDeploymentAndWait = k8sutil.UpdateDeploymentAndWait
 func (c *Cluster) Start() error {
 	logger.Infof("start running mgr")
 
-	var dashboardPort int
-	if c.dashboard.Port == 0 {
-		// select default ports
-		if c.cephVersion.Name == cephv1.Luminous {
-			dashboardPort = dashboardPortHttp
-		} else {
-			dashboardPort = dashboardPortHttps
-		}
-	} else {
-		// crd validates port >= 0
-		dashboardPort = c.dashboard.Port
-	}
-
 	for i := 0; i < c.Replicas; i++ {
-		if i >= len(mgrNames) {
-			logger.Errorf("cannot have more than %d mgrs", len(mgrNames))
+		if i >= 2 {
+			logger.Errorf("cannot have more than 2 mgrs")
```

Comment:
Why is that? Because of the dashboard?

Reply:
I think Rook actually only supports 1 mgr. @travisn. The old `len(mgrNames)` was `len([]string{"a", "b"}`, so the expression hasn't changed.

Reply:
Historically, Rook originally had two mgr daemons, but then after discussions we decided to go down to a single mon. We preserved the log to spin up two mons in case the decision is reversed. With K8s keeping the pod running or restarting if it fails, it didn't seem necessary to start another one.

Reply:
Thanks, this looks acceptable.